desktop-widgets/printerwebengine.cpp

```diff
@@ -46,8 +46,10 @@ void Printer::onLoadFinished()
 					  img.style.width = \"100%\"; \
 					  profile.appendChild(img); \
 					} \
-				", [this](const QVariant &v) { emit profilesInserted(); });
+				", [](const QVariant &v) {;});
```

Comment:
? Why would you pass a callback that does nothing ?
There should be a non-callback version of that call - if the API forces you to use a callback I would think that it is crucial to do something in there.